src/panels/config/cloud/dialog-cloud-already-connected/dialog-cloud-already-connected.ts

```diff
@@ -115,20 +141,25 @@ class DialogCloudAlreadyConnected extends LitElement {
       css`
         ha-dialog {
           --mdc-dialog-max-width: 535px;
-        }
-        .instance-details {
-          display: flex;
-          flex-direction: column;
-          margin-right: 20%;
-        }
-        .instance-detail {
-          display: flex;
-          flex-direction: row;
-          justify-content: space-between;
-        }
-        p {
-          margin-top: 0;
-          margin-bottom: 12px;
+
+          .instance-details {
+            display: flex;
+            flex-direction: column;
+            margin-right: 92px;
+            padding-bottom: 2.5em;
+
+            .instance-detail {
+              display: flex;
+              flex-direction: row;
+              justify-content: space-between;
+
+              .obfuscated {
+                display: flex;
+                flex-direction: row;
+                margin-right: -48px;
+              }
```

Comment:
Please don't use css nesting. Earliest support is iOS 17, we want to keep iOS 12 support

Reply:
why safari have to ruin all the fun :(